bundles/org.palladiosimulator.dataflow.confidentiality.analysis/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Data Flow Confidentiality Analysis
+Bundle-SymbolicName: org.palladiosimulator.dataflow.confidentiality.analysis
+Bundle-Version: 1.0.0.qualifier
+Automatic-Module-Name: org.palladiosimulator.dataflow.confidentiality.analysis
+Bundle-RequiredExecutionEnvironment: JavaSE-17
+Require-Bundle: org.eclipse.core.runtime;bundle-version="3.24.0",
+ org.eclipse.core.resources;bundle-version="3.16.0",
+ tools.mdsd.library.standalone.initialization;bundle-version="0.2.0",
+ tools.mdsd.library.standalone.initialization.log4j;bundle-version="0.2.0",
+ tools.mdsd.library.standalone.initialization.emfprofiles;bundle-version="0.2.0",
+ org.junit.jupiter.api;bundle-version="5.8.1",
```

Comment:
We should think whether we actually want these test dependencies in the main bundle. Normally we have a separate test bundle in Palladio. However, I assume this is for the workaround with the profiles.